packages/inventory/package.json

```diff
@@ -18,7 +18,6 @@
     "@openmsupply-client/config": "^0.0.0",
     "@openmsupply-client/system": "^0.0.0",
     "react": "^17.0.2",
-    "react-dom": "^17.0.2",
-    "react-router-dom": "^6.0.2"
+    "react-dom": "^17.0.2"
```

Comment:
React 18 has entered beta now - might be a good idea to try an upgrade..

Reply:
that's a yup! although a slightly nervous one.. that could be another bigjobs.